core/parachain/availability/recovery/recovery_impl.cpp

```diff
@@ -229,14 +355,20 @@ namespace kagome::parachain {
   void RecoveryImpl::done(
       Lock &lock,
       ActiveMap::iterator it,
-      const std::optional<outcome::result<AvailableData>> &result) {
-    if (result) {
-      cached_.emplace(it->first, *result);
+      const std::optional<outcome::result<AvailableData>> &result_op) {
+    if (result_op.has_value()) {
+      auto &result = result_op.value();
+
+      cached_.emplace(it->first, result);
+
+      if (result.has_value()) {
+        // incFullRecoveriesFinished("unknown", "success"); // TODO fix strategy
```

Comment:
Create issue or remove

Reply:
It will fixed in my next PR (systematic chunks)